### ISSUE TITLE
[1.18.2] `NullPointerException: Cannot invoke "net.minecraft.client.resources.sounds.SoundInstance.m_7904_()" because the return value of "net.minecraftforge.client.event.sound.PlaySoundEvent.getSound()" is null`

### ISSUE THREAD

**Issue by SettingDust** [TheComputerizer/Music-Triggers#103]
**Describe the bug**
```
java.lang.NullPointerException: Cannot invoke "net.minecraft.client.resources.sounds.SoundInstance.m_7904_()" because the return value of "net.minecraftforge.client.event.sound.PlaySoundEvent.getSound()" is null
	at mods.thecomputerizer.musictriggers.client.ClientEvents.playSound(ClientEvents.java:54) ~[musictriggers-1.18.2-6.1.jar%23212!/:?] {re:classloading,pl:rei_plugin_compatibilities:B,pl:runtimedistcleaner:A}
	at net.minecraftforge.eventbus.ASMEventHandler_366_ClientEvents_playSound_PlaySoundEvent.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%232!/:?] {}
	at net.minecraftforge.client.ForgeHooksClient.playSound(ForgeHooksClient.java:384) ~[forge-1.18.2-40.2.1-universal.jar%23315!/:?] {re:mixin,re:classloading,pl:rei_plugin_compatibilities:B,pl:mixin:A}
	at net.minecraft.client.sounds.SoundEngine.m_120312_(SoundEngine.java:326) ~[client-1.18.2-20220404.173914-srg.jar%23310!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.sounds.SoundManager.m_120367_(SoundManager.java:229) ~[client-1.18.2-20220404.173914-srg.jar%23310!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B}
	at net.minecraft.client.multiplayer.ClientLevel.m_7785_(ClientLevel.java:452) ~[client-1.18.2-20220404.173914-srg.jar%23310!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,pl:mixin:APP:dynamiclightsreforged.mixins.json:ClientWorldMixin,pl:mixin:APP:rubidium.mixins.json:features.chunk_rendering.MixinClientWorld,pl:mixin:APP:rubidium.mixins.json:features.fast_biome_colors.MixinClientWorld,pl:mixin:APP:mixins.oculus.vertexformat.json:block_rendering.MixinClientLevel,pl:mixin:APP:krypton.mixins.json:client.fastchunkentityaccess.ClientWorldMixin,pl:mixin:APP:starlight.mixins.json:client.world.ClientLevelMixin,pl:mixin:APP:mixins.sndctrl.json:MixinClientWorld,pl:mixin:APP:flywheel.mixins.json:ClientLevelMixin,pl:mixin:APP:entityculling.mixins.json:ClientWorldMixin,pl:mixin:APP:blue_skies.mixins.json:ClientLevelMixin,pl:mixin:APP:betterbiomeblend.mixins.json:MixinClientWorld,pl:mixin:APP:architectury.mixins.json:MixinClientLevel,pl:mixin:APP:blueprint.mixins.json:client.ClientLevelMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.player.LocalPlayer.m_5496_(LocalPlayer.java:470) ~[client-1.18.2-20220404.173914-srg.jar%23310!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:itemphysic:dropItem,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,xf:fml:itemphysic:dropItem,pl:mixin:APP:wildbackport-common.mixins.json:client.LocalPlayerMixin,pl:mixin:APP:supplementaries.mixins.json:LocalPlayerMixin,pl:mixin:APP:fastload.mixins.json:client.ClientPlayerEntityMixin,pl:mixin:APP:ars_nouveau.mixins.json:elytra.ClientElytraMixin,pl:mixin:APP:konkrete.mixin.json:client.IMixinLocalPlayer,pl:mixin:APP:travelerstitles.mixins.json:LocalPlayerTickMixin,pl:mixin:APP:create.mixins.json:HeavyBootsOnPlayerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at com.yungnickyoung.minecraft.travelerstitles.render.TitleRenderManager.updateDimensionTitle(TitleRenderManager.java:151) ~[TravelersTitles-1.18.2-Forge-2.1.1.jar%23285!/:1.18.2-Forge-2.1.1] {re:mixin,re:classloading,pl:rei_plugin_compatibilities:B}
	at com.yungnickyoung.minecraft.travelerstitles.render.TitleRenderManager.playerTick(TitleRenderManager.java:87) ~[TravelersTitles-1.18.2-Forge-2.1.1.jar%23285!/:1.18.2-Forge-2.1.1] {re:mixin,re:classloading,pl:rei_plugin_compatibilities:B}
	at net.minecraft.client.player.LocalPlayer.handler$cop000$onPostPlayerTick(LocalPlayer.java:3618) ~[client-1.18.2-20220404.173914-srg.jar%23310!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:itemphysic:dropItem,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,xf:fml:itemphysic:dropItem,pl:mixin:APP:wildbackport-common.mixins.json:client.LocalPlayerMixin,pl:mixin:APP:supplementaries.mixins.json:LocalPlayerMixin,pl:mixin:APP:fastload.mixins.json:client.ClientPlayerEntityMixin,pl:mixin:APP:ars_nouveau.mixins.json:elytra.ClientElytraMixin,pl:mixin:APP:konkrete.mixin.json:client.IMixinLocalPlayer,pl:mixin:APP:travelerstitles.mixins.json:LocalPlayerTickMixin,pl:mixin:APP:create.mixins.json:HeavyBootsOnPlayerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.player.LocalPlayer.m_8119_(LocalPlayer.java:200) ~[client-1.18.2-20220404.173914-srg.jar%23310!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:itemphysic:dropItem,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:rei_plugin_compatibilities:B,xf:fml:itemphysic:dropItem,pl:mixin:APP:wildbackport-common.mixins.json:client.LocalPlayerMixin,pl:mixin:APP:supplementaries.mixins.json:LocalPlayerMixin,pl:mixin:APP:fastload.mixins.json:client.ClientPlayerEntityMixin,pl:mixin:APP:ars_nouveau.mixins.json:elytra.ClientElytraMixin,pl:mixin:APP:konkrete.mixin.json:client.IMixinLocalPlayer,pl:mixin:APP:travelerstitles.mixins.json:LocalPlayerTickMixin,pl:mixin:APP:create.mixins.json:HeavyBootsOnPlayerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
```

**To Reproduce**
Crashed in Dawncraft modpack

**Additional context**
Add any other context about the problem here. Remember to add the version of the game and mod you are using

**Comment by Cancel-Me** [comment]
I'm having exactly the same problem with the same modpack, unable to launch successfully. 

musictriggers-1.18.2-6.1.jar

**Comment by TheComputerizer** [comment]
This has been fixed in 6.2
